Sources/SwiftDocC/Infrastructure/Symbol Graph/SymbolGraphLoader.swift

```diff
@@ -146,6 +146,36 @@ struct SymbolGraphLoader {
         
         self.symbolGraphs = loadedGraphs.mapValues(\.graph)
         (self.unifiedGraphs, self.graphLocations) = graphLoader.finishLoading()
+        
+        unifiedGraphs.forEach { (_, unifiedGraph) in
+            var unifiedGraph = unifiedGraph
```

Comment:
fyi: this can be written more simply using a regular for loop
```suggestion
        for var unifiedGraph in unifiedGraphs.values {
```

Reply:
Nice, I didn't know about this, thx! https://github.com/apple/swift-docc/pull/807/commits/c55abc329895424275b96a3c47afef3f8c5bb3f9